### DISCUSSION TITLE
Very interested

### DISCUSSION THREAD

**Discussion by gavingrey** [Cryolite/kanachan#28] [General]
Hello, I am very interested in this project. I have an RTX 4090 that I would like to use for the cause of training. I attempted to use the mahjongsoul_sniffer repo to pull some data into S3 for training, however that is giving me some errors. Probably because I am using WSL 2 instead of native Linux. I am continuing trying to set that up and I look forwards to getting into training.

**Comment by Cryolite** [comment]
If you could tell me what command you ran and what error was output, I might be able to help in some way.

By the way, the easiest way to run programs in this repository on Windows is probably to install Docker Desktop for Windows. Note that Docker Desktop for Windows can coexist with Docker on WSL Linux.

**Comment by gavingrey** [comment]
I commented in one of the ongoing issues there
